pydeseq2/dds.py

```diff
@@ -1010,6 +1011,36 @@ def plot_dispersions(
             **kwargs,
         )
 
+    def plot_rle(
+        self,
+        normalize: bool = False,
+        save_path: Optional[str] = None,
+        **kwargs,
+    ):
+        """Plot ratio of log expressions for each sample.
+
+        Useful for visualizing sample to sample variation.
+
+        Parameters
+        ----------
+        normalize : bool, optional
+            Whether to normalize the counts before plotting. (default: ``False``).
+
+        save_path : str or None
+            The path where to save the plot. If left None, the plot won't be saved
+            (default: ``None``).
+
+        **kwargs
+            Keyword arguments for the scatter plot.
+        """
+        make_rle_plot(
+            count_matrix=self.X,
+            normalize=normalize,
+            sample_ids=self.obsm["design_matrix"].index,
+            save_path=save_path,
+            **kwargs,
```

Comment:
c.f. my comment in `utils.make_rle_plot`: if normalize, check whether we already have size factors in `self.obsm["size_factors")`